src/Symfony/Component/Routing/Generator/UrlGenerator.php

```diff
@@ -76,6 +76,11 @@ class UrlGenerator implements UrlGeneratorInterface, ConfigurableRequirementsInt
     );
 
     /**
+     * @var string This regexp matches all characters that are not or should not be encoded by rawurlencode (see list in array above).
+     */
+    protected $urlEncodingSkipRegexp = '#[^-.~a-zA-Z0-9_/@:;,=+!*|]#';
```

Comment:
Like @stof commented in the  other PR, this property must be private even if the other ones are protected (the rules for private vs protected changed since then).